File: src/main/java/com/grepp/curdsample/util/TimeFormatter.java
package com.grepp.curdsample.util;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class TimeFormatter {

    private static final DateTimeFormatter pattern = DateTimeFormatter.ofPattern("MM/dd/yyyy");

    public static LocalDate convertToLocalDate(String localDateStr) {
//        DateTimeFormatter pattern = DateTimeFormatter.ofPattern("MM/dd/yyyy");
        return LocalDate.parse(localDateStr, pattern);
    }

    public static String convertToStr(LocalDate localDate) {
        return localDate.format(pattern);
    }

    public static String convertToStr(LocalDateTime localDate) {
        return localDate.format(pattern);
    }

}

File: src/main/java/com/est/crudsample/util/TimeFormatter.kt
package com.est.crudsample.util

import java.time.LocalDate
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

private val pattern: DateTimeFormatter = DateTimeFormatter.ofPattern("MM/dd/yyyy")

fun convertToLocalDate(localDateStr: String): LocalDate {
    return LocalDate.parse(localDateStr, pattern)
}

fun convertToStr(localDate: LocalDate): String {
    return localDate.format(pattern)
}

fun convertToStr(localDate: LocalDateTime): String {
    return localDate.format(pattern)
}